### ISSUE TITLE
6.1 Accessibility Update Please: Can you permanently code in a plus mark that stays at the top and makes the whole line clickable?

### ISSUE THREAD

**Issue by a4jp-com** [WordPress/gutenberg#44451]
### Description

Can you permanently code in a plus mark that stays at the top under the title and it always just adds a block to the top as its function please?

There should always be a plus mark under the wp-block-post-title. When you first add a block there is a plus mark at the top and then it gets clicked it goes missing after adding the first block. The other plus marks are all okay when you mouse over the area to add blocks. Just the top one is missing. This means we have to duplicate blocks and move them up every time instead of just do a one click add a new block.

block-editor-block-list__insertion-point-indicator should also be clickable. The target click area is only 24px so making the whole line difficult to click. If it were clickable it would be better for accessibility and people using custom mice due to physical disabilities, etc. Or even better make the the whole rectangular div clickable as even the line is pretty thin.

Gutenberg should be more user friendly with changes like these. Especially before 6.1 comes out. Gutenberg is the main block adding interface for WordPress now. If it's bad, it is one of the reasons why people don't wanna use WordPress which is actually  a really nice CMS to use compared to others. I can see Gutenberg's user ranking is really low now (with over 2300 people giving it a 1 star ranking). I'd maybe give it a 2 star ranking as it works but 3 says good and it isn't yet. That is in its current form, as just having bit like add go missing isn't good. Please make the GUI of Gutenberg more user friendly. 

![Screen Shot 09-26-22 at 04 50 PM](https://user-images.githubusercontent.com/9621044/192223293-d0ce0305-ac3b-4019-91d9-f49a7179792b.PNG)
![Screen Shot 09-26-22 at 04 52 PM 002](https://user-images.githubusercontent.com/9621044/192223172-b7212b7d-3c13-41bf-9d53-2c8c7790cb37.PNG)

![Screen Shot 09-26-22 at 04 52 PM 001](https://user-images.githubusercontent.com/9621044/192223266-6ca004aa-871f-4863-90c9-4c9af6ba36e9.PNG)
![Screen Shot 09-26-22 at 04 52 PM](https://user-images.githubusercontent.com/9621044/192223280-c25ebaea-8703-474c-a142-4d33a545ea72.PNG)

### Step-by-step reproduction instructions

Create a new page. Add a block using the plus mark in the content area. Move to the top of the block to add a new block. There is no coded plus mark. This should be coded so it is just always there as we need to add content to the top of the page often, especially for some types of websites. I have to add blocks to the top of page 3 to 5 times a day now.

### Environment info

WordPress Version 6.0.2
Gutenberg Version 14.1.1

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by a4jp-com** [comment]
Thank you @carolinan

**Comment by a4jp-com** [comment]
This is still a problem.

**Comment by a4jp-com** [comment]
Can the plus code be permanently put under the title area? Then each block that shows up just needs the plus code at the bottom.
